test/elections.spec.ts

```diff
@@ -27,6 +27,7 @@ import {
     stakedEvents,
     stakeChangedEvents
 } from "./event-parsing";
+import {totalmem} from "os";
```

Comment:
I'm curious about the story behind this unused import :)

Reply:
removing this and other unused imports